sources/logdog/src/commands.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+/// Provides the list of command that `logdog` will run.
+
+/// Returns the standard list of `logdog` commands.
+pub(crate) fn commands() -> Vec<(&'static str, &'static str)> {
+    vec![
```

Comment:
```suggestion
pub(crate) fn commands() -> &[(&'static str, &'static str)] {
    &[
```
Will Rust let you get away without the Vec if you change the types here and in the callers?